assets/src/stories-editor/plugins/export.js

```diff
@@ -0,0 +1,57 @@
+/* global fetch, alert */
+
+/**
+ * WordPress dependencies
+ */
+import { PluginMoreMenuItem } from '@wordpress/edit-post';
+const { ampStoriesExport, FormData, URL } = window;
+
+const handleExport = () => {
+	const fromData = new FormData();
+
+	// Add the form data.
+	fromData.append( 'action', ampStoriesExport.action );
+	fromData.append( '_wpnonce', ampStoriesExport.nonce );
+	fromData.append( 'post_ID', document.getElementById( 'post_ID' ).value );
+
+	// Request the export.
+	fetch( ampStoriesExport.ajaxUrl, {
+		method: 'POST',
+		body: fromData,
+	} )
+		.then( ( response ) => {
+			if ( response.ok ) {
+				// Handle the returned blob data.
```

Comment:
Might be worth adding a snackbar notification upon successful download.

https://developer.wordpress.org/block-editor/components/snackbar/

Reply:
@swissspidy How would we know if the download was successful or failed?

Reply:
Or if the user canceled the download?

Reply:
Le me rephrase: after clicking on the button, a snackbar notification can be added so the user gets some feedback.

Just like when you click on "Copy All Content" in the Gutenberg block editor.